packages/bpk-component-modal/src/BpkModalV2/BpKModal.module.scss

```diff
@@ -50,8 +65,12 @@
 
   .bpk-modal {
     position: fixed;
+    top: 50%;
+    left: 50%;
     display: block;
     z-index: 1;
+    transform: translate(-50%, -50%) scale(1);
```

Comment:
Still need to center as Safari 14 and 15 does support the dialog element. So need to center it